autotest.pm

```diff
@@ -147,13 +142,13 @@ sub runalltests {
                 }
             }
             else {
-                if ($snapshots_supported && $flags->{milestone}) {
+                if ($snapshots_supported && ($flags->{milestone} || $bmwqemu::vars{MAKETESTSNAPSHOTS})) {
                     make_snapshot('lastgood');
```

Comment:
hmm...that means if MAKETESTSNAPSHOTS=1 all snapshot tag name(qemu-img snapshot -l) will be lastgood, am I right? I thought the goal of MAKESNAPSHOTS is user can get a snapshot with clear snapshot point(name) what they can jump to the place they want, this change sounds not a good idea to me...